homeassistant/components/qbittorrent/translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Device is already configured"
+        },
+        "error": {
+            "invalid_auth": "Invalid authentication",
+            "cannot_connect": "Invalid server URL given"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "username": "Username",
+                    "password": "Password",
+                    "url":"Server URL"
```

Comment:
Was this file manually edited instead of generated via the `python3 -m script.translations develop` automation? The missing space on this line is a suspicious hint of a manual edit.

Reply:
it was manually created yes, I've corrected it now

Reply:
👍 . The same situation applies to the other two language JSON files. I'm not sure how or if those should be checked in without going through the real localization process.